model/user.go

```diff
@@ -115,13 +114,49 @@ type LearningGroup struct {
 	// The display name of the private channel for this learning group (Channels.DisplayName)
 	ChannelDisplayName string `json:"channel_display_name"`
 
+	// An array of LearningGroupMember structs for every member of this learning group (channel)
+	Members []LearningGroupMember `json:"members"`
+
+	// Denotes whether or not the current user has left this learning group
+	HasLeftGroup bool `json:"has_left_group"`
+}
+
+// This is a struct for a learning group query row (ex. capstone)
+// **Note: Using pointers as property types allows for null values
+type LearningGroupQueryRow struct {
+	// The name of the learning group (the channel name minus the learning group type's prefix)
+	//     ex. plg-30 => 30
+	LearningGroupName string `json:"learning_group_name"`
+
+	// The prefix for this learning group type (personal channel name from the config)
+	//     ex. plg
+	LearningGroupPrefix string `json:"learning_group_prefix"`
+
+	// The id of the private channel for this learning group
+	ChannelId string `json:"channel_id"`
+
+	// The slug name of the private channel for this learning group (Channels.Name)
+	ChannelSlugName string `json:"channel_slug_name"`
+
+	// The display name of the private channel for this learning group (Channels.DisplayName)
+	ChannelDisplayName string `json:"channel_display_name"`
+
 	// A comma delimited list of the usernames of the members of this learning group (channel)
 	Members *string `json:"members"`
```

Comment:
I believe you've actually change the value in Members (and commented on it elsewhere:
```
// Note: in a learning group query row, members is a double delimited string of users (; delimited) and
// each user's id and username (, delimited)
```